docs/current-1.1/tools/dotnet-nuget-delete.md

```diff
@@ -0,0 +1,81 @@
+---
+title: dotnet-nuget-delete command | .NET Core SDK
+description: The dotnet-nuget-delete command deletes or unlists a package from the server. 
+keywords: dotnet-nuget-delete, CLI, CLI command, .NET Core
+author: karann-msft
+ms.author: mairaw
+manager: wpickett
+ms.date: 11/10/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 6ddffde4-c789-4e90-990e-d35f6a6565d4
+---
+
+#dotnet-nuget-delete
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-delete -- Deletes or unlists a package from the server. 
+
+## Synopsis
+
+`dotnet nuget delete [<package_name> <package_version>] 
+    [--help] [--source] [--non-interactive] 
+    [--api-key] [--force-english-output] [--verbosity] [--config-file]`
+
+## Description
+
+The `dotnet nuget delete` command deletes or unlists a package from the server. For NuGet.org, the action is to unlist the package.
+
+## Options
+
+`-h|--help`
+
+Prints out a short help for the command.  
+
+`-s|--source <SOURCE>`
+
+Specifies the server URL. Supported URL's for nuget.org include `http://www.nuget.org`, `http://www.nuget.org/api/v3` and `http://www.nuget.org/api/v2/package`. 
+For private feeds, substitute the host name (for example, `%hostname%/api/v3`).
+
+`--non-interactive`
+
+Does not prompt for user input or confirmations.
+
+`-k|--api-key <API_KEY>`
+
+The API key for the server.
+
+`--force-english-output`
+
+Forces command-line output to be in English.
+
+`--verbosity`
+
```

Comment:
For your question in the email, --verbosity can have three possible values - normal, quiet, detailed